### PR TITLE
Upstream animation-reverse-end-exclusive.html from Blink

### PR DESCRIPTION
**Pull request by alancutter**
Import https://chromium.googlesource.com/chromium/src/+/3d473d69a3f7fd0d6cb0b01cf2e9d72c4d068b6f/third_party/WebKit/LayoutTests/web-animations-api/animation-reverse-end-exclusive.html

### CODE REVIEW

File: web-animations/timing-model/animations/reversed-active-internal-boundaries.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Reversed animation active interval bounds</title>
+<link rel="help" ­href="???">
```
Comment (suzyh) [inline web-animations/timing-model/animations/reversed-active-internal-boundaries.html:5]: Add a TODO to fix this or mention it in the TODO below; this "???" could be easily forgotten.

File: web-animations/timing-model/animations/reversed-active-internal-boundaries.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      'Reversed animation is not in effect when current time is 0');
+
+  animation.currentTime = 1;
+  assert_equals(animation.effect.getComputedTiming().progress, 1,
+      'Reversed animation is in effect when current time is end time');
+
+}, "Reversed animations flip the inclusive/exclusive boundaries of their effect's active interval");
+
+</script>
+</body>
```
Comment (suzyh) [inline web-animations/timing-model/animations/reversed-active-internal-boundaries.html]: If I interpret this correctly, github is complaining about a lack of newline at the end of this file.

**Comment by suzyh** [comment]
lgtm with nits

File: web-animations/timing-model/animations/reversed-active-internal-boundaries.html (hunk 2 above)
Comment (alancutter) [inline web-animations/timing-model/animations/reversed-active-internal-boundaries.html]: Added newline.

File: web-animations/timing-model/animations/reversed-active-internal-boundaries.html (hunk 1 above)
Comment (alancutter) [inline web-animations/timing-model/animations/reversed-active-internal-boundaries.html:5]: Moved TODO.

**Pull request merged**